openedx/core/djangoapps/api_admin/models.py

```diff
@@ -35,6 +42,8 @@ class ApiAccessRequest(TimeStampedModel):
     reason = models.TextField(help_text=_('The reason this user wants to access the API.'))
     company_name = models.CharField(max_length=255, default='')
     company_address = models.CharField(max_length=255, default='')
+    base_url = models.CharField(max_length=255, default='')
```

Comment:
Do we actually need this, or can we use `[get_absolute_url](https://docs.djangoproject.com/en/1.9/ref/models/instances/#get-absolute-url)`?